apps/wallet-mobile/src/components/Button/Button.tsx

```diff
@@ -154,7 +154,6 @@ const useStyles = ({
     container: {
       backgroundColor,
       ...atoms.flex,
-      ...atoms.flex_grow,
```

Comment:
@jorbuedo I checked other buttons and it seems like this change has no negative implications. Maybe I'm missing something.

Reply:
Uff, I wouldn't be so sure. Had quite a bit of trouble trying to simplify that so that it worked for all cases, and couldn't get it right. What I do is when I don't want it to grow, wrap it in a View with whatever position you want.